jlm/util/Statistics.hpp

```diff
@@ -66,10 +71,19 @@ class Statistics {
 /**
  * Determines the settings of a StatisticsCollector.
  */
-class StatisticsCollectorSettings final {
+class StatisticsCollectorSettings final
+{
+  inline static const char* DefaultFilePath_ = "/tmp/jlm-stats.log";
```

Comment:
This breaks when running jlc in parallel, would it be possible to add a random string to the filename?

Reply:
yes, /tmp is where the resulting files ends up if nothing was specified. I do not like the working directory solution, as I would first need to find the working directory somehow, secondly it pollutes the working dir with the files (which might be unwanted by the user), and thirdly there is no automatic cleaning  up (which happens in /tmp for each reboot). I think /tmp is a reasonable default that does not interfere with the user if no one cares about the statistics.

Yes, running jlc in parallel will result in gibberish in the statistics files. I will address it in another PR.